lib/groonga-query-log/server-verifier.rb

```diff
@@ -181,8 +190,29 @@ def verify_command(groonga1_client, groonga2_client, command)
       command["output_type"] = "json"
       rewrite_filter(command, "filter")
       rewrite_filter(command, "scorer")
+
       response1 = groonga1_client.execute(command)
-      response2 = groonga2_client.execute(command)
+      # `groonga2` is new Groonga.
+      response2 = nil
+      request = groonga2_client.execute(command) do |response|
+        response2 = response
+      end
```

Comment:
↓みたいに`@options.verify_cancel?`じゃないときは既存のコードでよろしく。

```suggestion
      if @options.verify_cancel?
        response2 = nil
        request2 = groonga2_client.execute(command) do |response|
          response2 = response
        end
        # ...
      else
        response2 = groonga2_client.execute(command)
      end
```
```